### DISCUSSION TITLE
I want this very much the education but I am not enrolled at college

### DISCUSSION THREAD

**Discussion by 4115-source** [community/community#83060] [Other Feature Feedback, Questions, & Ideas]
### Hi everyone,

I want my brain to learn code and I want to further my education. But after reading what I have to do to enroll. Makes this opportunity impossible for me.

**Comment by RoyFlowers99** [comment]
I would begin by watching some YouTube tutorials. Learn HTML and CSS. Make a static site; you can serve it on here by using GitHub Pages. Once you feel confident writing basic HTML and CSS you should learn JavaScript. Build a small web app. There are a lot of great resources online to help you if you're passionate enough to invest your time in them. I'll link some of my favorites below as well as the links to Mozilla's Developer Network (MDN) website and W3 Schools. Good Luck!

<a href="https://youtu.be/G3e-cpL7ofc?si=48oErzC8Y796Ekbg">HTML and CSS Video Tutorial</a><br>
<a href="https://youtu.be/SBmSRK3feww?si=TLuvBv5avpl3-9FU">JavaScript Video Tutorial</a><br>
<a href="https://developer.mozilla.org/en-US/docs/Web/HTML">MDN Web Docs</a><br>
<a href="https://www.w3schools.com/">W3 Schools</a>

**Comment by UniversalCoderAJ** [comment]
I would suggest pick any technologies you interested in, start watching free videos online, practice every day and make sure you get certifications for what you have learned. It grows your career drastically by writing certifications you get direct interview calls for job offers.

**Comment by jpcchaves** [comment]
Don't give up! There's a lot of free content in the internet, what do you trying to learn? I mean, what technologies?
